cfgov/v1/tests/models/test_sublanding_page.py

```diff
@@ -0,0 +1,89 @@
+import mock
+
+from unittest import TestCase
+from v1.models import SublandingPage, BrowseFilterablePage, AbstractFilterPage
+from wagtail.wagtailcore.blocks import StreamValue
+
+import scripts._atomic_helpers as atomic
+from v1.tests.wagtail_pages import helpers
+
+import datetime as dt
+
+
+class SublandingPageTestCase(TestCase):
+    """
+    This test case checks that the browse-filterable posts of a sublanding page are
+    properly retrieved.
+    """
+    def setUp(self):
+        self.request = mock.MagicMock()
+        self.request.site.hostname = 'localhost:8000'
+        self.limit = 10
+        self.sublanding_page = SublandingPage(title='title')
+
+        helpers.publish_page(child=self.sublanding_page)
+        self.post1 = BrowseFilterablePage(title='post 1')
+        self.post2 = BrowseFilterablePage(title='post 2')
+        # the content of this post has both a full_width_text and a filter_controls
+        self.post1.content = StreamValue(self.post1.content.stream_block,
+                                         [atomic.full_width_text, atomic.filter_controls], True)
+        # this one only has a filter_controls
+        self.post2.content = StreamValue(self.post1.content.stream_block,
+                                         [atomic.filter_controls], True)
+
+        helpers.save_new_page(self.post1, self.sublanding_page)
+        helpers.save_new_page(self.post2, self.sublanding_page)
+
+        # manually set the publication date of the posts to ensure consistent
+        # order of retrieval in test situations, otherwise the `date_published`
+        # can vary due to commit order
```

Comment:
Curious about this comment. Is it the order that varies? Or just the fact that the dates will be the same because these are all published at the same time?